redis_benchmarks_specification/__cli__/stats.py

```diff
@@ -338,9 +338,6 @@ def generate_stats_cli_command_logic(args, project_name, project_version):
                     logging.warn("Unable to detect group in {}".format(cmd))
 
         priority_list = sorted(((priority[cmd], cmd) for cmd in priority), reverse=True)
-        priority_list_usecs = sorted(
```

Comment:
@markovamaria why was this removed?

Reply:
> @markovamaria why was this removed?
I agree that it is separate issue. :) But it blocked CI for this commit